tools/src/main/scala/scala/scalanative/build/Platform.scala

```diff
@@ -0,0 +1,9 @@
+package scala.scalanative.build
+
+object Platform {
+  private lazy val osUsed = System.getProperty("os.name", "unknown").toLowerCase
```

Comment:
This won't work on a Turkish/Lithuanian/Azeri system.
```suggestion
  private lazy val osUsed = System.getProperty("os.name", "unknown").toLowerCase(Locale.ROOT)
```